deps/rabbit/src/rabbit_channel.erl

```diff
@@ -106,6 +106,7 @@
           consumer_timeout,
           authz_context,
           %% defines how ofter gc will be executed
```

Comment:
This line is referring to `writer_gc_threshold`, let's move it around accordingly and keep all comments on the previous line.